install.md

```diff
@@ -8,15 +8,14 @@
 sudo pacman -S podman
 ```
 
-If you have problems when running podman in [rootless](README.md#rootless) mode follow [these instructions](https://wiki.archlinux.org/index.php/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
+If you have problems when running Podman in [rootless](README.md#rootless) mode follow [these instructions](https://wiki.archlinux.org/index.php/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
```

Comment:
Maybe it's just me, but when I first read this when it's resolved to a web page, I missed the link and was looking on this page.  Maybe s/these instructions/the instructions here/ and have the word "here" be the link?